prExplorer/client-app/src/app/common/form/CustomSelectInput.tsx

```diff
@@ -0,0 +1,26 @@
+import { useField } from 'formik';
+import React from 'react';
+import { Form, Label, Select } from 'semantic-ui-react';
+
+interface CustomSelectInputProps {
+    placeholder: string;
+    name: string;
+    options: any;
+    label?: string;
+}
+
+export default function CustomSelectInput(customSelectInputProps: CustomSelectInputProps) {
+    const [field, meta, helpers] = useField(customSelectInputProps.name);
+    return (
+        <Form.Field error={meta.touched && !!meta.error}>
+            <label>{customSelectInputProps.label}</label>
+            <Select clearable options={customSelectInputProps.options} value={field.value || null} onChange={(e, d) => helpers.setValue(d.value)}
```

Comment:
@aliyorulmazdev are you sure you're using prettier in VS Code? Your code does not seem to be formatted